tests/tools/test_util.yml

```diff
@@ -0,0 +1,107 @@
+
+- name: download_miniwdl
+  tags:
+    - miniwdl
+    - util
+  command: >-
+    miniwdl run -d test-output/. -i tests/tools/input_json/util_download.json --task download tools/util.wdl
+  stdout:
+    contains_regex:
+      - '"download.downloaded_file": ".*test-output/out/downloaded_file/license.txt"'
+  files:
+    - path: test-output/out/downloaded_file/license.txt
+      md5sum: cf3575bd84ab3151c7e9700b5f1a9746
+      contains:
+        - "MIT License"
+
+- name: get_read_groups_miniwdl
```

Comment:
OOS for this PR, but just wondering "why is this task in `util.wdl` instead of `samtools.wdl`"?

Tangent: I'm not a fan of how long either of these files are (`util` and `samtools`). I'd consider breaking both of them up into more specific or specialized files. The length of these files is making it annoying to find things IMO.

Reply:
The underlying task has some special logic on how it formats the output. It seems more appropriate to be in `util`, because it is manipulating the `samtools` output so much. The reformatting portion is also `star`-specific.

```
        if ~{format_for_star}; then
            samtools view -H ~{bam} \
                | grep "@RG" \
                | cut -f 2- \
                | sed -e 's/\t/ /g' \
                | awk '{print}' ORS=' , ' \
                | sed 's/ , $//' > read_groups.txt
        else
            samtools view -H ~{bam} | grep "@RG" > read_groups.txt
        fi
```